src/main/resources/data/miningmaster/advancements/recipes/forging/ultima_axe_4.json

```diff
@@ -2,29 +2,17 @@
   "parent": "minecraft:recipes/root",
   "rewards": {
     "recipes": [
-      "miningmaster:forging/ultima_axe_4"
+      "miningmaster:forging/ultima_axe_5"
```

Comment:
Why did you change this to 5 in this file, the 4 file, but delete the 5 file?

Reply:
That's... A good question! I don't know! I deleted the 5 file due to the loss of the Garnet Axe, I think, but I don't know why I made this odd mistake here.